### ISSUE TITLE
请问怎么看预测结果为真人音频还是合成视频呢

### ISSUE THREAD

**Issue by Judy19811** [xieyuankun/Codecfake#7]
结果文件中，这个分数怎么看才是真，多少为假呢
LA_E_4162943 1.299897434137165e-07 spoof
LA_E_9711562 1.2908270008438194e-08 spoof
LA_E_1873564 1.8409064068691805e-06 spoof
LA_E_4906555 4.19502119792316e-16 spoof
LA_E_3396772 3.4175545974093836e-27 spoof
LA_E_9845174 1.2106907026492038e-18 spoof
LA_E_9234337 0.00011092011845903471 spoof
LA_E_1183624 0.008070427924394608 spoof
LA_E_4790567 4.472591186299724e-09 spoof

**Comment by xieyuankun** [comment]
您好，第二列是输出的logits，趋向0代表假，趋向1代表真。
因为该列取得值是`F.softmax(w2v2_outputs)[:, 0]`，取的logits的第0维对应是真实标签的位置，所以该分数趋向1代表判断正确（真），趋向0代表真实标签判断错误（假）。
最后一列是标签。
